src/transformers/tokenization_bert.py

```diff
@@ -606,7 +606,7 @@ def __init__(
         mask_token="[MASK]",
         clean_text=True,
         tokenize_chinese_chars=True,
-        strip_accents=True,
+        strip_accents=None,
```

Comment:
The option to strip accents is still available, but not specified by default. If not specified, the behavior is the same than the default implementation.